scripts/Engineering/gui/engineering_diffraction/tabs/calibration/presenter.py

```diff
@@ -7,20 +7,32 @@
 # pylint: disable=invalid-name
 from __future__ import (absolute_import, division, print_function)
 
+from Engineering.gui.engineering_diffraction.tabs.common import INSTRUMENT_DICT
 from mantidqt.utils.asynchronous import AsyncTask
 from mantid.simpleapi import logger
+from mantidqt.utils.observer_pattern import Observable
 
 
 class CalibrationPresenter(object):
     def __init__(self, model, view):
         self.model = model
         self.view = view
-        self.view.set_on_calibrate_clicked(self.on_calibrate_clicked)
         self.worker = None
+        self.calibration_notifier = self.CalibrationNotifier(self)
+
+        self.current_calibration = {"vanadium_path": None, "ceria_path": None}
```

Comment:
For the future is it worth considering making this its own type? Something like 

```python
class CalibrationInfo
    vanadium_path
    ceria_path
```
It can be clearer to have first-class types with named fields as the IDE can autocomplete and refactoring is generally easier to perform automatically.

Reply:
Yes, this makes more sense than a dictionary, there are a few improvements like this that could stand to be in a separate PR or just go in the next feature addition.